python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -76,15 +89,21 @@ def data_cat_3():
     return cat1, cat2
 
 
-@pytest.mark.parametrize("data", [data_cat_2()])
-@pytest.mark.xfail(raises=AttributeError)
-def test_categorical_accessor_initialization(data):
+@pytest.mark.parametrize("data", [data_cat_1()])
+def test_categorical_accessor_initialization1(data):
     sr = Series(data.copy())
     dsr = dgd.from_cudf(sr, npartitions=5)
     dsr.cat
 
 
-@pytest.mark.xfail(reason="")
+@pytest.mark.parametrize("data", [data_cat_2()])
+def test_categorical_accessor_initialization2(data):
+    sr = Series(data.copy())
+    dsr = dgd.from_cudf(sr, npartitions=5)
+    with pytest.raises(AttributeError):
+        dsr.cat
```

Comment:
here as well

Reply:
The data in `dsr` is of not Datetime type. It's a Int Series here.